templates/common/_base/files/NetworkManager-keyfiles.yaml

```diff
@@ -0,0 +1,6 @@
+mode: 0644
+path: "/etc/NetworkManager/conf.d/02-keyfiles.conf"
```

Comment:
The only other thought I've had since we talked is that maybe we want to leave a gap in the numbering just in case for some reason there would be a need to insert another file between these two. I can't imagine why we would, but it doesn't cost anything to make this 10 or 20 instead of 2 and it could save us some grief someday.

Reply:
Yes, makes much sense, changed